cdap-tms/pom.xml

```diff
@@ -42,10 +42,6 @@
       <version>${project.version}</version>
     </dependency>
 
-    <dependency>
```

Comment:
Do you also want to remove it from https://github.com/caskdata/cdap/blob/develop/cdap-examples/Purchase/pom.xml#L45-L48?

Reply:
There's also a dependency on the common-http module in other places, such as in the `cdap-clients` module.